### ISSUE TITLE
Gem differs from plugin... issues with gem.

### ISSUE THREAD

**Issue by keilmillerjr** [brendanlim/mobile-fu#20]
Hello. I am running rails 3.0.7.
## If I install mobile-fu via add to my gemfile:

```
gem 'mobile-fu'
```

When I view the page on the iPad (or with an iPad user agent):
- The stylesheet tag for mobile.css is automatically added to application.mobile.erb.
- _The stylesheet tag for mobile_ipad.css *_IS NOT*\* automatically added to application.mobile.erb*

When I view the page on the iPhone (or with an iPhone user agent):
- The stylesheet tag for mobile.css is automatically added to application.mobile.erb.
- _The stylesheet tag for mobile_ipad.css *_IS*\* automatically added to application.mobile.erb*
## If I install mobile-fu via plugin:

```
rails plugin install git://github.com/brendanlim/mobile-fu.git
```

When I view the page on the iPad (or with an iPad user agent):
- The stylesheet tag for mobile.css is automatically added to application.mobile.erb.
- _The stylesheet tag for mobile_ipad.css *_IS*\* automatically added to application.mobile.erb*

When I view the page on the iPhone (or with an iPhone user agent):
- The stylesheet tag for mobile.css is automatically added to application.mobile.erb.
- _The stylesheet tag for mobile_ipad.css *_IS*\* automatically added to application.mobile.erb*
## Bottom line

All works correctly via installing by plugin, and not the gem. This leads me to believe that the gem and plugin are two different versions, and that there is a bug or difference in release in the gem.

Please look into this and fix if possible (and let us know that it is fixed!). With Rails 3, it is nice to have everything installed via gemfile and bundler.

Thank you,

Keil

**Comment by benlangfeld** [comment]
The release gem is from my fork, not from this repo which is unmaintained. Can you please refile this issue on my fork?

**Comment by brenttheisen** [comment]
@benlangfeld I don't think your fork has the issues functionality in Github enabled.

**Comment by benlangfeld** [comment]
It does now

**Comment by benlangfeld** [comment]
This project is abandoned. Please see the active fork at http://github.com/benlangfeld/mobile-fu. Please test with the released gem and master branch of the new home for the project, and file an issue on the other repo if you still have problems.

See #40.
